frontend/src/metabase/admin/people/events.js

```diff
@@ -0,0 +1,5 @@
+export const LOAD_MEMBERSHIPS = "metabase/admin/people/LOAD_MEMBERSHIPS";
```

Comment:
We store action names in `actions.js` file or in the main "duck" file, e.g. `people.js`.

Reply:
I had to move these into another file so that I could import them into `entities/group.js` without introducing a cyclic dependency. unfortunately, I'm not able to create the actions in here either since I need `services` to do that, which was what caused the dependency issue. I could rename this file to `actions.js` though if you think that would be helpful